applications/webhooks/src/webhooks_disabler.erl

```diff
@@ -72,6 +73,43 @@ find_failures() ->
     Keys = wh_cache:fetch_keys_local(?CACHE_NAME),
     find_failures(Keys).
 
+-spec flush_hooks(wh_json:objects()) -> non_neg_integer().
+flush_hooks(HookJObjs) ->
+    lists:sum(
+      [flush_failures(
+         wh_doc:account_id(HookJObj)
+         ,wh_doc:id(HookJObj)
+        )
+       || HookJObj <- HookJObjs
+      ]
+     ).
+
+-spec flush_failures(ne_binary()) -> non_neg_integer().
+-spec flush_failures(ne_binary(), api_binary()) -> non_neg_integer().
+flush_failures(AccountId) ->
+    flush_failures(AccountId, 'undefined').
```

Comment:
I was going to expose this in webhooks_maintenance so you could flush failures per account. Thanks for reminding me to expose it!